portfolio/src/main/webapp/assets/script.js

```diff
@@ -26,3 +26,20 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function collapseMenu() {
+    var i;
+    var elements = document.getElementsByClassName("collapsible");
+
+    for (i = 0; i < elements.length; i++) {
+        elements[i].addEventListener("click", function() {
+            this.classList.toggle("active");
+            var content = this.nextElementSibling;
+            if (content.style.maxHeight) {
+                content.style.maxHeight = null;
+            } else {
+                content.style.maxHeight = content.scrollHeight + "px";
+            }
```

Comment:
This might be easier to read with a ternary operator:

`content.style.maxHeight = (content.style.maxHeight ? null :  content.scrollHeight + "px");`